routes/admin.js

```diff
@@ -1,21 +1,18 @@
 const express = require('express')
 const router = express.Router()
 const passport = require('passport')
+const db = require('../database/db')
 
-router.get('/', function(request, response){
-  request.url < 2 ? response.render('admin') : response.redirect('/loggedout')
+router.post('/', function(request, response){
+  let { name, description, available, quantity, img } = request.body
```

Comment:
ALWAYS `const` OVER `let` to avoid possible bugs from re-assignment, when you really intend to have a constant value.